test/e2e-tests.sh

```diff
@@ -48,10 +49,35 @@ TEST_YAML_IGNORES=${TEST_YAML_IGNORES:-""}
 # test for example "s2i"
 TEST_TASKRUN_IGNORES=${TEST_TASKRUN_IGNORES:-""}
 
+# Define this variable if you want to run all tests and not just the modified one.
+TEST_RUN_ALL_TESTS=${TEST_RUN_ALL_TESTS:-""}
```

Comment:
We need to update the [catalog periodic](https://github.com/tektoncd/plumbing/blob/master/prow/config.yaml#L1175) to set this environment variable, right ? If yes, we should do it ahead of time as it is won't affect the current run.

Reply:
Yep, let me make sure this works tho before remving the DNM 👍